methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<p>I changed the first part of this cell to this for running on the OSL:</p><pre class="ql-syntax" spellcheck="false"># top-level directory for calval work
calval_dir = '/home/jovyan/calval/'
# magic path object
os.chdir('/home/jovyan')
calval_dir_obj = Path.cwd()/'calval'
</pre><p>I think it is better to set the top-level directory instead of running from wherever the notebook happens to be.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I would avoid to habe the hard coded paths in it.</p><p>However, do agree with eric we should make like a work_folder to seperate it from the notebook</p>

Reply:
<p>suggesting to keep it as cwd/'work'/'coseimsic'/ =&gt; sitenames.</p><p>Same for the other notebooks cwd/'work'/'secular'/ =&gt; sitenames</p><p> </p><p>In this way we can make it clear to reviewers where the procesisng is happening</p>

Reply:
<p>We may want to avoid this to be more generalized and accessible. E.g., if people run this from their own remote server, we want to avoid having it write to the home disk. If a user wishes to, they can hardcode the workdirectory as they please in the prep sections</p>